pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/Common.java

```diff
@@ -193,7 +193,7 @@ public static String removingLeadingDot(String text) {
 	 * @return File object for java file
 	 */
 	public static File getJavaFile(File srcDir, String javaPackage, String className) {
-		File packagePath = new File(srcDir.getPath() + File.separatorChar + javaPackage.replaceAll("\\.",File.separator));
+		File packagePath = new File(srcDir.getPath() + File.separatorChar + javaPackage.replaceAll("\\.","\\" + File.separator));
```

Comment:
I don't see why this has "\\"+File.seperator

Reply:
The second argument in `replaceAll` is also a regex. In case of `/` it works just fine, but if it's `\` the method call fails with `IllegalArgumentException` because it interprets it as a missing character that was supposed to be escaped.